corehq/apps/domain/templates/domain/internal_calculations.html

```diff
@@ -39,8 +39,8 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
-    <div class="form form-horizontal">
+{% block page_content %}
+    <div class="form-horizontal">
         <p><button type="button" id="load-all-btn" class="btn btn-primary">{% trans 'Load EVERYTHING' %}</button></p>
         <hr />
         {% for calc_tag in order %}
```

Comment:
@kkrampa can you please take a look at http://getbootstrap.com/migration/
This is the first link in the migration guide and covers all the css classes that need to be renamed. For instance, you forgot to rename `control-group` to `form-group` or `controls-row` to `row`, all input classes to `form-control` and then wrap the corresponding fields in controls inside separate div tags. eg

```
<div class="controls row">
<div class="col-sm-8">
  ...field data...
</div>
<div class="col-sm-4">
 ...load property button...
</div>
</div>
```
I'll clarify the migration guide now to make sure this step isn't missed in the future.